bom/runtime/pom.xml

```diff
@@ -1137,6 +1138,11 @@
                 <artifactId>javax.ws.rs-api</artifactId>
                 <version>${javax.ws.rs-api.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.testng</groupId>
+                <artifactId>testng</artifactId>
+                <version>${testng.version}</version>
+            </dependency>
```

Comment:
No, we need it at `compile` time for `test-framework/arquillian`

Reply:
We're not really testing anything with it, it's there just to be able to write a testng arq. extension

Reply:
I thought you could override the scope.

So if in `depMan` it was `test`, in the other pom it could be `compile`? My worry is someone forgets to add `test` if they add it somewhere else

Reply:
So should it be in `build-parent` instead of the `bom`?

Reply:
Yes, you can override the scope.
I am not sure about where it should be, going through the setup quarkus is using, I ended up putting the version here, no dep here declares test scope though.